icons.js

```diff
@@ -0,0 +1,547 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
Sure thing. But since we don't have any other kinds of icons, I thought of a simpler name. :)